woocommerce/woocommerce
# Delete woocommerce-gutenberg-products-block.php file

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

`woocommerce-gutenberg-products-block.php` was only executed when WooCommerce Blocks was run as a feature plugin. Since the merge in the monorepo, it should be no longer needed.

### How to test the changes in this Pull Request:

Testing this PR means smoke testing WooCommerce Blocks. For example:

1. Open the post editor.
2. Add the Mini-Cart and Product Collection blocks (or any other blocks you want).
3. Verify they can be added.
4. Publish the post.
5. Verify they show up correctly in the frontend.
6. Try adding a product to the cart and verify the Mini-Cart updates.

### Changelog entry


<details>

#### Significance


#### Type


#### Message 

#### Comment

Delete woocommerce-gutenberg-products-block.php file, which was no longer executed since the merge of WC Blocks into the monorepo

</details>
